ansible/roles/cephmetrics-common/tasks/setup_repos.yml

```diff
@@ -2,37 +2,11 @@
 - name: Enable EPEL
   yum:
     name: "https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ ansible_distribution_major_version }}.noarch.rpm"
-    state: "{{ 'present' if use_epel else 'absent' }}"
+    state: present
   when:
     - ansible_pkg_mgr == "yum"
     - devel_mode
 
-- name: Install cephmetrics development repo
-  copy:
-    src: files/cephmetrics-devel.repo
-    dest: /etc/yum.repos.d/cephmetrics.repo
-  when:
-    - ansible_pkg_mgr == "yum"
-    - not use_epel
-    - devel_mode
-
-- name: Remove cephmetrics repo
-  file:
-    path: /etc/yum.repos.d/cephmetrics.repo
-    state: absent
-  when:
-    - ansible_pkg_mgr == "yum"
-    - use_epel
-    - devel_mode
-
-- name: Remove old cephmetrics production repo
```

Comment:
Shouldn't we keep this at least for production deployments?

Reply:
I don't think anyone that actually deployed RHCS would have had this .repo installed; in fact RHCS would have removed it, even if pre-release deploy code had been run in the past.

Reply:
iirc, we deployed this repo before we started shipping with brew and errata. It is probably ok to remove it here but we should add it to the purge.yml.

Reply:
I have just checked and purge.yml already removes it so we are ok to merge.